### ISSUE TITLE
minor encoding issue in base::order unit tests

### ISSUE THREAD

**Issue by jangorecki** [Rdatatable/data.table#3004]
there seems to be issue, not in data.table unit tests, but `base::order` unit test on my machine.
```
Running test id 1590.13      Test 1590.13 ran without errors but failed check that x equals y:
> x = base::order(c(x2, x1, x1, x2)) 
First 6 of 4 (type 'integer'): [1] 1 4 2 3
> y = 1:4 
First 6 of 4 (type 'integer'): [1] 1 2 3 4
Mean relative difference: 0.4444444
Running test id 1590.17      Test 1590.17 ran without errors but failed check that x equals y:
> x = base::order(c(x2, x1, x1, x2)) 
First 6 of 4 (type 'integer'): [1] 2 3 1 4
> y = INT(1, 4, 2, 3) 
First 6 of 4 (type 'integer'): [1] 1 4 2 3
Mean relative difference: 0.4

Warning message:
In parse(n = -1, file = file, srcfile = NULL, keep.source = FALSE) :
  invalid input found on input connection '/home/jan/git/tmplib/data.table/tests/tests.Rraw'
```
```
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.4 LTS

Matrix products: default
BLAS: /usr/lib/libblas/libblas.so.3.6.0
LAPACK: /usr/lib/lapack/liblapack.so.3.6.0

locale:
[1] C

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.5.1
> Sys.getlocale()
[1] "C"
```
```
$ locale
LANG=en_IN
LANGUAGE=en_IN:en
LC_CTYPE="en_IN"
LC_NUMERIC="en_IN"
LC_TIME="en_IN"
LC_COLLATE="en_IN"
LC_MONETARY="en_IN"
LC_MESSAGES="en_IN"
LC_PAPER="en_IN"
LC_NAME="en_IN"
LC_ADDRESS="en_IN"
LC_TELEPHONE="en_IN"
LC_MEASUREMENT="en_IN"
LC_IDENTIFICATION="en_IN"
LC_ALL=
```

**Comment by jangorecki** [comment]
If I remove my `~/.Renviron` file:
```
LC_ALL=C
```
I am not getting those failures anymore. Which is very similar issue to #2771.

**Comment by jangorecki** [comment]
resolved by 5eebdad76521ae10762f141a95178df33673accb
